dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -1110,6 +1110,19 @@ public void waitForSSLRestart() throws Exception {
 
     }
 
+    /**
+     * Wait for the server to state that it is listening on its SSL port
+     *
+     * @throws Exception
+     */
+    public void waitForSSLStart() throws Exception {
+        //wait for "CWWKO0219I: TCP Channel defaultHttpEndpoint-ssl has been started and is now listening for requests on host"
+        String sslStartMsg = waitForStringInLogUsingMark("CWWKO0219I:.*defaultHttpEndpoint-ssl.*");
+        if (sslStartMsg == null) {
+            Log.warning(c, "SSL may not have started properly - future failures may be due to this");
+        }
```

Comment:
It would be much more helpful to throw an exception here so that we don't even try to run the test if the SSL port didn't open. That way we get the actual problem that occurred in the test failure exception, rather than a later consequence of the problem.

That said, I do note that the sibling `waitForSSLRestart` method merely logs.

I don't want to make a method that just logs (because it's not useful) but I also don't want to have a similar looking method that does something different (because it's confusing). I would move this method back into our project where we can do what's sensible for our tests and make it throw an exception.